### PR TITLE
Fix ably-echo import

### PR DESCRIPTION
**Pull request by sacOO7**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -25,12 +26,11 @@ window.Echo.connector.ably.connection.on(stateChange => {
...
 });
 ```
-You can set custom [clientOptions](https://ably.com/docs/api/realtime-sdk?lang=javascript#client-options) when creating an `Echo` instance.
+You can set additional ably-js [clientOptions](https://ably.com/docs/api/realtime-sdk?lang=javascript#client-options) when creating an `Echo` instance.
 
 ```
     broadcaster: 'ably',
-    authEndpoint: 'http://www.localhost:8000/broadcasting/auth'
-      // Additional ably specific options - https://ably.com/docs/api/realtime-sdk?lang=javascript#client-options  
+    authEndpoint: 'http://www.localhost:8000/broadcasting/auth', // absolute or relative url to laravel-server
```
Comment (qsdstefan) [inline .github/README.md:33]: Is there any particular reason why we're providing `www.localhost:8000` in the example and not just `localhost:8000` which is the default address when using `php artisan serve`?

Comment (sacOO7) [reply .github/README.md:33]: It's to denote that we can provide either `absolute` or `full url` ( I have already added the comment). 
Also, if UI server is hoted on different IP than laravel server, there would be need to provide a separate IP. 
So `localhost` is just a placeholder for that IP. 
`relative` url can work most of the times, but `absolte` url will also be needed during certain times.

**Review by qsdstefan — APPROVED** [review]
LGTM!

**Review by owenpearson — APPROVED** [review]
LGTM, thanks

**Pull request merged**